.editorconfig

```diff
@@ -0,0 +1,55 @@
+# EditorConfig is awesome: https://EditorConfig.org
+# If you are using VS Code consider using the following extension
+# https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
```

Comment:
Can you include the url to the umbrella project, so that if people update this file, they would update a file there too (which is still missing, but soon).